aiida/orm/implementation/django/node.py

```diff
@@ -488,9 +488,9 @@ def store_all(self, with_transaction=True):
                 "Node with pk= {} was already stored".format(self.pk))
 
         # For each parent, check that all its inputs are stored
-        for label in self._inputlinks_cache:
+        for link in self._inputlinks_cache.itervalues():
```

Comment:
Small nitpick: ``itervalues`` is Python 2 - only. To be Py3 compatible, it should be ``six.itervalues``: http://python-future.org/compatible_idioms.html#iterating-through-dict-keys-values-items

But when we port to Py3 we will anyway have to fix many of these things, so it's not really crucial to do it now.